store/src/main/java/org/apache/rocketmq/store/MappedFileQueue.java

```diff
@@ -34,6 +37,7 @@
     private static final InternalLogger LOG_ERROR = InternalLoggerFactory.getLogger(LoggerName.STORE_ERROR_LOGGER_NAME);
 
     private static final int DELETE_FILES_BATCH_MAX = 10;
+    private static final Pattern MAPPED_FILENAME_PATTERN = Pattern.compile("^\\d{20}$");
```

Comment:
In my opinion,the number '20' which in pattern 'MAPPED_FILENAME_PATTERN' and in `org.apache.rocketmq.common.UtilAll#offset2FileName`'s `nf.setMinimumIntegerDigits(20);` is a magic number.Maybe they can be replaced by a named constant.

Reply:
Good catch!

I just pushed a new changeset to centralize these magic numbers in the UtilAll class like:

    public static final int MAPPED_FILENAME_DIGITS = 20;
    public static final Pattern MAPPED_FILENAME_PATTERN = Pattern.compile("^\\d{20}$");

I don't reduce instances of magic number '20' to one since I don't want string-concated-regex, which is ugly and hard to read.

Reply:
Hi,
   I had fixed the issue as commented a few days ago. Can have a second look at the pull request?

  Thanks!